src/Survey.jl

```diff
@@ -2,6 +2,7 @@ module Survey
 
 using DataFrames
 using Statistics
+import Statistics: quantile
```

Comment:
We are already importing quantile in line 7.

Reply:
Statistics and Statsbase have different definitions of `quantile` fn